reana_server/scheduler.py

```diff
@@ -83,6 +92,10 @@ def doesnt_exceed_max_reana_workflow_count():
             .scalar()
         )
         if running_workflows >= REANA_MAX_CONCURRENT_BATCH_WORKFLOWS:
+            logging.info(
+                "Workflow could not be started due to reached upper limit on "
```

Comment:
We can print the detected values for easier deduction, such as:

Cannot start workflow 60515eff-33ca-468a-a55c-6cf72d72af5d because there are already 42 running workflows whilst the allowed maximum is 25.